src/dlgprefreplaygaindlg.ui

```diff
@@ -202,7 +248,20 @@
       <item row="4" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
-         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not change the ReplayGain of playing tracks.</string>
+         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not apply the ReplayGain to playing tracks. </string>
+        </property>
+        <property name="wordWrap">
+         <bool>true</bool>
+        </property>
+       </widget>
+      </item>
+      <item row="0" column="0">
+       <widget class="QLabel" name="label_8">
+        <property name="text">
+         <string>The normal average ReplayGain playback level is -14 dB. This offers enough headroom for the dynamic peaks of almost all tracks. However it might be not enough for mixing or adding effects without clipping or it might sound to quiet for modern pop music that is normalized to 0 db and compressed to an average playback level of -7 dB or more.  So you may tweak the Initial Boost to solve these issues.</string>
```

Comment:
>>    practically the default ReplayGain level already offers a mixing headroom.

> I still don't agree with this -- we are using replaygain to help level out the tracks, not apply headroom. It's only an > accident of replaygain's intended purpose that it applies headroom. As I've argued I think it makes more sense to > boost replaygained tracks so they match non-replaygain tracks, then have a master headroom.

Sorry for beeing not precise enough. I agree with your view, this pain is not focusing the mixing headroom.  But the initial sentence is still correct, because in general all multipliers at any place in the engine just allow to set a headroom. And if there is one, it is up to the user how to use it.